tests/data/test_rsa_key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDP3CkiWRD4gJqz
+977fVzCaofo/49/BUYU3rt8MVJCMpSnHT7aeM95KDbQYiY4s6mPf9cO/6JjHvbmQ
+7xPNTnnhRruA0BWL768kXw5rETijmlA/WIJMbkLgwxtGafZYn3b1o2mqSWKxK690
+KynynUv7sydxabjnk4nJVxC4nHAp1VI8X/DawIXbBTzu4ktPmBXKzriRNZjtk+D1
+ZIkaIQonP1Wpzse6kg/6wW3OhO5BHwA4fCINwGjBzqr0mmBN5ry9nBIpff48uXNV
+9gVIIgQRzwLgNgWFfnXOHS77+0n0z0BtmUatyeAhREwqGcMoA5aivUatHf5rEbaC
+Kjr3WSGNAgMBAAECggEAaEFN5Gt15p5iedfORakuuLB7myYJaaYgwlAUkdOseM6y
+iMSDnQ/4832yEgiUZhTeKUvUdeINF0oi0/4GGZi96u8WRsKXvAto9j4zNiJ6HRze
+GRekqv82zhMuEAA/zi2VdhkTe5S5SpHVu9eWf5jDrqgqJWlYk9Zdar4fpejZHTF4
+M+6EE6eUX76CWoqwUMLAEj/aCEM2qeYHWk1VQyX83HB1jy+BX2cEDeZvxmV7P7mp
+ZlZ6gEOdVfhlewZJbC+SWRwm5B+dawscaBRE4pJtiaMV7qOmz17GpfgE/RpJlqVA
+fe790HdQdCIKIBeKTpqCyP7iWCrDu3cjoSd44LrpBQKBgQD+0ti3m9fsuNkNDuC+
+dgAXkR8YGvAQe8M9bcmACBFGGiLStCrpSgZwagDuta2mZLZT31lG4KGb5MrOV0zz
+Q64YhglRaSHPtqi2ITJHxX9RdHTSj9w6axFWUvfXHbEZDI7E4daMcgQZ6u9dr7VF
+DFHchW7OPJjozeokqMG1mpQ3rwKBgQDQ0c/exV8Vu0M7Iteu01rhvJxtWSx0xWv6
+B1bgyYOkyWpYVKdIk1LcVAZhgvtbiikJTlXKgwttQMgdygPpTUiwPInB/NCF0oAs
+orfpGeBexLFJFvbN3CoNajGmDSHx6RQi9K1V842hjPBA+b6uvEbfUlpyH4AP21PM
+EfjNMo9NgwKBgDOO3aJouct/qwrlU7u1jFc4WZ469Q+guuQW7oolF7mjWCBhq7z/
+6UWdbQrfX38nKWzW5+1bTdeI9y/AoiUmMHdtxzzdlKW+Q2x2UwIKh7QnZ+uih+Ca
+ASwCJXs02rxCujBDsXFBMGs+CahfAMIzt+xyYvT/dcDEyPcZ3fesiwipAoGBAIRS
+v++BkJxbuuGpVZVSFz/+Xf2oyVQBmkepCPOOnp34iCwLEKobuSEnGZgHATLjnNdp
+zVFzsvT7XRQLZGkdcRdEdWL4ykZSuqgOQI40uIo1B8ayB5kxj3BKv8VigwUhVoJE
+G+bgW/poLgJuf9eINTzkma3BqkviBvrE1K1rAYXzAoGBAL+98/XzKDyhw0ncJASv
+SQ+3+WOoeotDX6AHS5PVDNj/r2wJspA6D82/GIfVi58DexhzgjVYrFi0Fi+00DyG
+3fVnuAeJCdVuqNEpW7N8QZGvvavPOBPJj20m0bcjZ7w0+4fnpT6fjF8Q3JOQTdTa
+iLVX9N4wZxsVH9pPVjNWWKLy
+-----END PRIVATE KEY-----
```

Comment:
Same questions as above.

Usually putting private key in the repo is not a good idea. Instead, generate them dynamically on each test.

Reply:
Same as above.